### PR TITLE
base navigation, default screens are added

### PR DESCRIPTION
**Pull request by zhmurko-anastasi**

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import * as React from 'react';
```
Comment (serhiyburkow) [inline App.tsx:1]: Great! Take care for the future :)

File: navigation/index.tsx [hunk 2]
```diff
@@ -1,32 +1,85 @@
...
+      <Stack.Screen name="NotFound" component={NotFoundScreen} options={{headerShown: false  }} />
+      <Stack.Screen name="Root" component={BottomTabNavigator} options={{ headerShown: false }} />
     </Stack.Navigator>
   );
 }
 
+const BottomTab = createBottomTabNavigator<RootTabParamList>();
+
+function BottomTabNavigator() {
+  const getIconColor = (focused: boolean) => focused ? mainBlack : mainGrey;
```
Comment (serhiyburkow) [inline navigation/index.tsx:46]: You can make it simpler way while you don't need additional conditions: 
`
<Tab.Navigator screenOptions={() => ({
            tabBarActiveTintColor: 'tomato',
            tabBarInactiveTintColor: 'gray',
        })}
`

File: src/modules/CreateScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Pressable, StyleSheet, Text, View } from 'react-native';
+
+import { RootStackScreenProps } from '../../types';
+
+export default function CreateScreen({ navigation }: RootStackScreenProps<'Home'>) {
+  return (
+    <View style={styles.container}>
+      <Pressable onPress={() => navigation.replace('Login')} style={styles.link}>
```
Comment (serhiyburkow) [inline src/modules/CreateScreen.tsx:9]: I suggest you to create common component for button.

File: types.tsx [hunk 4]
```diff
@@ -14,8 +14,16 @@ declare global {
 }
 
 export type RootStackParamList = {
+  Root: NavigatorScreenParams<RootTabParamList> | undefined;
```
Comment (serhiyburkow) [inline types.tsx:17]: Always good to use enum for navigation names to explicitly avoid "silent" typos and to make possible refactoring easier. 

```
    [Routes.mainStack]: undefined;
    [Routes.dashboard]: undefined;

```

**Review by serhiyburkow — CHANGES_REQUESTED** [review]

File: navigation/index.tsx [hunk 5]
```diff
@@ -1,32 +1,85 @@
...
+function BottomTabNavigator() {
+
+  return (
+    <BottomTab.Navigator initialRouteName={Routes.home} screenOptions={{tabBarActiveTintColor: mainBlack, tabBarInactiveTintColor: mainGrey,}} >
+      <BottomTab.Screen
+        name={Routes.home}
+        component={HomeScreen}
+        options={{
+          title: 'Home',
+          tabBarIcon: ({ color }: { color: any }) => <Ionicons color={color}  size={30} name="home" />,
```
[the remark this answers is not in the document]
Comment (zhmurko-anastasi) [reply navigation/index.tsx:55]: Done

**Pull request merged**